### PR TITLE
Add Gradle coordinates for the REST client

### PR DESCRIPTION
**Pull request by dadoonet**

### CODE REVIEW

**Comment by dadoonet** [comment]
@javanna Could you review this please?

File: docs/java-rest/usage.asciidoc [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.elasticsearch.client%22[Maven
 Central]. The minimum Java version required is `1.7`.
 
+The low-level REST client is subject to the same release cycle as
+elasticsearch. Replace the version with the desired client version, first
+released with `5.0.0-alpha4`. There  is no relation between the client version
+and the elasticsearch version that the client can communicate with. The
+low-level REST client is compatible with all elasticsearch versions.
+
+=== Maven coordinates
```
Comment (javanna) [inline docs/java-rest/usage.asciidoc]: maybe these two new sections can stay in the same page as Maven Central Repository? did you try generating the updated docs and see what they look like?

File: docs/java-rest/sniffer.asciidoc [hunk 2]
```diff
@@ -8,7 +8,14 @@ Nodes Info api and uses jackson to parse the obtained json response.
 
 Compatible with Elasticsearch 2.x and onwards.
 
-=== Maven Repository
+The low-level REST client is subject to the same release cycle as
```
Comment (javanna) [inline docs/java-rest/sniffer.asciidoc]: This change (Removing the title) moves this section to the previous page I think. Is this what you wanted to do? Should we have the same type of maven central introduction that we have for the rest client in the other docs page?

**Review by javanna — APPROVED** [review]
Left two nits, thanks for the improvement though

**Comment by rjernst** [comment]
Can we replace "Coordinates" with "Configuration" or "Sample Configuration" for both maven and gradle section names? There is no such thing as a "gradle coordinate".

**Comment by javanna** [comment]
> Can we replace "Coordinates" with "Configuration" or "Sample Configuration" for both maven and gradle section names? There is no such thing as a "gradle coordinate"

Not a big deal but I thought the terminology is correct: https://maven.apache.org/pom.html#Maven_Coordinates . You mean that it doesn't apply to gradle? But after all those are "gradle maven coordinates" no? :)

**Comment by rjernst** [comment]
Yes Maven Coordinates exist, but that is not what we have there. The coordinates are the combination of groupId, artifactId and version. But what we have there is example xml to place in a pom.xml file. And likewise, the gradle being added is gradle configuration to *use* the maven coordinates, just like the xml example.

File: docs/java-rest/sniffer.asciidoc [hunk 3]
```diff
@@ -22,11 +29,17 @@ Add the following to your `pom.xml` file:
...
-supports fetching the nodes list from elasticsearch 2.x and onwards.
+=== Gradle coordinates
+
+Here is how you can configure the dependency using gradle as a dependency manager.
+Add the following to your `build.gradle` file:
+
+["source","groovy",subs="attributes"]
+--------------------------------------------------
+dependencies {
+    compile group: 'org.elasticsearch.client', name: 'sniffer', version: '{version}'
```
Comment (rjernst) [inline docs/java-rest/sniffer.asciidoc]: You can use the short form:
`compile 'org.elasticsearch.client:sniffer:{version}'`

File: docs/java-rest/usage.asciidoc (hunk 1 above)
Comment (dadoonet) [inline docs/java-rest/usage.asciidoc]: Agreed. I did not notice that chunk size is 1 for this repo. So I'll put that text back on both pages.

**Comment by dadoonet** [comment]
Here is how it now looks like. Let me know what you think.

maven repository java rest client elastic google chrome aujourd hui at 17 00 23

**Review by rjernst — APPROVED** [review]
LGTM, thanks @dadoonet!

**Comment by dadoonet** [comment]
Argh. I forgot to change also the sniffer page... Will push a new commit soonish.

**Pull request merged**